### ISSUE TITLE
Old version of YUI compressor

### ISSUE THREAD

**Issue by drwlrsn** [benschwarz/bonsai#34]
Hi, it kinda looks like this project is dead, but I thought you and those looking at this project should know that the version of the YUI Compressor used has issues with media queries. This means that when you bonsai --cultivate everything looks peachy keen, but when you go to bonsai --repot the YUI Compressor breaks your fancy responsive website.  

See http://yuilibrary.com/projects/yuicompressor/ticket/2527986 for an explanation.

TL;DR YUI Compressor needs to be updated.

**Comment by mariuscoto** [comment]
I find this project very useful and I don't see why you have stopped developing it.

As for the issues, I had none what so ever. I would personally check and repair
the code, but I am afraid I don't understand Ruby so well…sorry.

Marius.

**Comment by benschwarz** [comment]
@drwlrsn I'm a little unsure about your diagnosis of a dead project, but whatever — thanks for the heads up on YUI compressor. I'll update it. 

@cmarius02 I stopped active development on bonsai almost 3 years ago, I built it for a billable project of mine, made it feature complete… and then stopped. Since then I've made numerous bug fixes and a few features that were suggested by users, those features were particularly well thought out by those users and matched my ideals for bonsai. I'm really glad you've had no issues, and if you do — please log them, I'll check it out.

**Comment by drwlrsn** [comment]
I'm sorry.  I shouldn't have jumped to conclusions. I just saw there wasn't much activity. I can create a pull request, but if I can suggest that maybe it might be best to use the YUI Compressor gem instead of including the compressor itself.  That way you don't have to update it yourself.  I'm not much of a ruby person or I would give you a much larger hand.

Drew

**Comment by benschwarz** [comment]
HI both, I've made some changes to use the gem edition of YUI compressor (and dropped jruby support in the process)
Are you able to give the gem a once over run before I push it to rubygems?

**Comment by drwlrsn** [comment]
Absolutely. I can test it in the next 24 hours and get back to you. 

On 2013-01-19, at 8:11 PM, Ben Schwarz notifications@github.com wrote

**Comment by mariuscoto** [comment]
Could you please be more specific, please ? I don't really understand how to help you.

Marius.

**Comment by benschwarz** [comment]
@cmarius02, don't worry, sounds like @drwlrsn is going to test.
If you wanted to help test, you'd need to download the source and install the gem from there, but really—don't panic if you don't know how to.

**Comment by drwlrsn** [comment]
Everything looks good on my end.  If you want to double check you can bonsai with something like Skeleton or another responsive scaffold. Lately all my favourite projects are written in Ruby. Perhaps it's time to start drinking the sweet sweet Kool-Aid. Again, thanks for this lovely project.

**Comment by benschwarz** [comment]
Thanks so much for verifying this Drew, I'll push the gem release shortly.
